src/Compilers/Core/Portable/Operations/ControlFlowGraphBuilder.cs

```diff
@@ -5631,6 +5631,11 @@ internal override IOperation VisitPlaceholder(IPlaceholderOperation operation, i
             return new PlaceholderExpression(operation.PlaceholderKind, semanticModel: null, operation.Syntax, operation.Type, operation.ConstantValue, IsImplicit(operation));
         }
 
+        public override IOperation VisitConversion(IConversionOperation operation, int? captureIdForResult)
```

Comment:
It seems there is no change to the method. Consider reverting the change, since only moving the method.